packages/fast-components-styles-msft/README.md

```diff
@@ -119,6 +119,293 @@ const StyleSheet = {
+- `lines`: The logical number of lines the component takes, 1 by default
+
+```JavaScript
+import { heightNumber } from "@microsoft/fast-components-styles-msft";
+
+const inputSize: DesignSystemResolver<string> = `${heightNumber() / 2}`
+);
+
+const styles: ComponentStyles<ClassNameContract, DesignSystem> = {
+    styled_component: {
+        width: inputSize,
+        height: inputSize
+    },
+};
+```
+
+#### densityCategorySwitch
+
+Returns a value based on the higher-level category for the density setting. Use this function to adjust things like type size and sizing that is based on a category rather than individual density.
+
+**Parameters**:
+- `compactValue`: The adjustment when the category is "compact"
+- `normalValue`: The adjustment when the category is "normal"
+- `spaciousValue`: The adjustment when the category is "spacious"
+ */
+
+```JavaScript
+import { densityCategorySwitch } from "@microsoft/fast-components-styles-msft";
+
+const styles: ComponentStyles<ClassNameContract, DesignSystem> = {
+    styled_component: {
+        marginRight: `${densityCategorySwitch(4, 8, 12)}px`
+    },
+};
+```
+
+#### horizontalSpacing
+
+Returns the standard horizontal spacing typically used for text and icons formatted in the provided unit or px by default.
+
+**Parameters**:
+- `adjustment`: Any border that should be removed from the overall content spacing; 0 by default
+- `unit`: The optional unit of measurement; px by default
+
+```JavaScript
+import { horizontalSpacing } from "@microsoft/fast-components-styles-msft";
+
+const styles: ComponentStyles<ClassNameContract, DesignSystem> = {
+    styled_component: {
+        paddingLeft: horizontalSpacing(2),
+        paddingRight: horizontalSpacing(2),
+    },
+};
+```
+
+#### horizontalSpacingNumber
+
+Returns the standard horizontal spacing for text and icons as a number.
+
+**Parameters**:
+- `adjustment`: Any border that should be removed from the overall content spacing; 0 by default
+
+```JavaScript
+import { horizontalSpacingNumber } from "@microsoft/fast-components-styles-msft";
+
+const glyphWidth: number = 16;
+
+const styles: ComponentStyles<ClassNameContract, DesignSystem> = {
+    styled_component: {
+        marginRight: `${horizontalSpacingNumber(-2) + glyphWidth}px`
+    },
+};
+```
+
+#### glyphSize
+
+Returns the width and height for a glyph formatted in pixels.
+
+**Parameters**:
+- `designSystem` | `unit`: Accepts either a design system configuration or a string
+
+```JavaScript
+import { glyphSize } from "@microsoft/fast-components-styles-msft";
+
+const styles: ComponentStyles<ClassNameContract, DesignSystem> = {
+    styled_component: {
+        width: glyphSize,
+        height: glyphSize,
+    },
+};
+```
+
+### Disabled
+
+#### applyDisabledState
+
+Used to apply disabled styles to an element.
+
+```JavaScript
+import { applyDisabledState } from "@microsoft/fast-components-styles-msft";
+
+const styles: ComponentStyles<ClassNameContract, DesignSystem> = {

```

Comment:
This is marked as "JavaScript" but is actually TypeScript since you are providing types.